recipe/common.php

```diff
@@ -113,14 +115,6 @@
 });
 
 /**
- * Default arguments and options.
- */
-argument('stage', InputArgument::OPTIONAL, 'Run tasks only on this server or group of servers');
-option('tag', null, InputOption::VALUE_OPTIONAL, 'Tag to deploy');
-option('revision', null, InputOption::VALUE_OPTIONAL, 'Revision to deploy');
-option('branch', null, InputOption::VALUE_OPTIONAL, 'Branch to deploy');
-
-/**
```

Comment:
I moved this section into TaskCommand. I have no idea why they are called here. @elfet do you have any thoughts here?